js/apps/account-ui/test/account-security.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { test, expect } from "@playwright/test";
+import { login } from "./login";
+
+test.describe("Account security page", () => {
+  test("Check linked accounts available", async ({ page }) => {
+    await login(page, "jdoe", "jdoe", "photoz");
+    await page.getByRole("button", { name: "Account security" }).click();
+    const element = await page.$(
+      'li.pf-c-nav__item[data-ouia-component-id="OUIA-Generated-NavItem-8"]',
```

Comment:
can't we have a better locator then this?

Reply:
I agree, any way we can improve this?

Reply:
Thanks @edewit and @jonkoops I have now improved this test. For some strange reason, I was not able to access the element via `page.getByTestId("account-security/linked-accounts");` but this is now possible. Let's review and merge this. Thanks!